tests/integrations/flax/train_test.py

```diff
@@ -0,0 +1,24 @@
+from tango.common.logging import initialize_logging, teardown_logging
+from tango.common.testing import TangoTestCase
+
+
+class TestTrainStep(TangoTestCase):
+    def setup_method(self):
+        super().setup_method()
+        initialize_logging(enable_cli_logs=True)
+
+    def teardown_method(self):
+        super().teardown_method()
+        teardown_logging()
+
+    def test_trainer(self):
+        result_dir = self.run(
+            self.FIXTURES_ROOT / "integrations" / "flax" / "config.jsonnet",
+            include_package=[
+                "test_fixtures.integrations.common",
+                "test_fixtures.integrations.flax",
+            ],
+        )
+        assert (
+            result_dir / "train" / "work" / "checkpoint_state_latest" / "checkpoint_0"
+        ).is_file()
```

Comment:
Log

```
+ pytest -v tests/integrations/flax
============================= test session starts ==============================
platform linux -- Python 3.9.13, pytest-7.1.2, pluggy-1.0.0 -- $PREFIX/bin/python
cachedir: .pytest_cache
rootdir: $SRC_DIR
plugins: flaky-3.7.0
collecting ... collected 6 items

tests/integrations/flax/data_test.py::TestDataStep::test_dataloader PASSED [ 16%]
tests/integrations/flax/data_test.py::TestDataStep::test_sample_data PASSED [ 33%]
tests/integrations/flax/format_test.py::TestTorchFormat::test_read_write PASSED [ 50%]
tests/integrations/flax/optim_test.py::test_all_optimizers_registered PASSED [ 66%]
tests/integrations/flax/optim_test.py::test_all_lr_schedulers_registered PASSED [ 83%]
/home/conda/feedstock_root/build_artifacts/tango-split_1659720967230/test_tmp/run_test.sh: line 7:  2805 Killed                  pytest -v tests/integrations/flax
Tests failed for tango-flax-0.11.0-py39hf3d152e_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING:conda_build.build:Tests failed for tango-flax-0.11.0-py39hf3d152e_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
TESTS FAILED: tango-flax-0.11.0-py39hf3d152e_0.tar.bz2
```

Reply:
Interestingly, it passes on OSX, so it's definitely not a question of the test being runnable at all. It's also not impossible that the segfault is coming from some aspect in the way that all the _many_ dependencies of `tango[flax]` are packaged in conda-forge, but it's still rather unusual.